packages/host/app/components/operator-mode/container.gts

```diff
@@ -61,37 +59,25 @@ export default class OperatorModeContainer extends Component<Signature> {
 
   constructor(owner: Owner, args: Signature['Args']) {
     super(owner, args);
-    // TODO remove this
-    (globalThis as any)._CARDSTACK_CARD_SEARCH = this;
```

Comment:
Good riddance!

Reply:
I think there is one last global used for invoking the card chooser from a card def. perhaps that can use the dynamic context as well....